doc/source/user_guide/tutorials/animate/animate_data.rst

```diff
@@ -0,0 +1,503 @@
+.. _ref_tutorials_animate_data:
+
+============
+Animate data
+============
+
+:bdg-warning:`MAPDL` :bdg-success:`LSDYNA`
```

Comment:
Add Fluent and CFX

Reply:
See older comment about creating our own badges https://getbootstrap.com/docs/5.0/components/badge/#sass